src/admin/config.yml

```diff
@@ -436,6 +436,7 @@ collections:
       - {label: "Parent Page", name: "parentPageTitle", widget: "relation", collection: "projects", value_field: "title", search_fields: [title], required: false, hint: "This is required for sub-pages of a project to define the sub-page hierarchy."}
       - {label: "Subpage Order", name: "subPageOrder", widget: "number", required: false, hint: "Hint: if using this, all sub-pages at the same level should have an order number specified. Leave blank to have sub-pages appear in alphabetical order. "}
       - {label: "Thumbnail Image", name: "thumbnailImage", widget: "image", required: false, hint: "Suggested size: 680x494px. Images not fitting this aspect ratio will appear stretched."}
+      - {label: "Thumbnail Image Alt Text", name: "thumbnailAltText", widget: "string", required: false, hint: "Hint: thumbnail images will have alt text 'Thumbnail image for [Project name]' when this field is left empty."}
```

Comment:
```suggestion
      - {label: "Thumbnail Image Alt Text", name: "thumbnailAltText", widget: "string", required: false, hint: "If alt text is left blank, 'Thumbnail image for [Project Name]' will be used by default."}
```

Reply:
There are also a number of projects that have thumbnails that say "Thumbnail image for undefined" because the project does not have a Project Name (for example, Community-Led Co-design kit).

These issues can be fixed via the authoring interface after this gets merged. I'll file a separate issue to resolve those.

Reply:
> There are also a number of projects that have thumbnails that say "Thumbnail image for undefined" because the project does not have a Project Name (for example, Community-Led Co-design kit).
> 

@jhung, instead of using "Project Name" in the default thumbnail alt text, we could use "Body Title" to solve this issue as it's a required field. Normally project name and body title are identical. 

Moreover, shall we rename "Body Title" to "Title"?

Please let me know what you think. Thanks.

Reply:
@cindyli - it should be the project name used for the alt text. Body title can be longer and more descriptive which won't be appropriate for alt text.

I think "Body Title" should stay as it is because describes the location or function of that title (i.e. the title for the body of content). If it was just "title" there may be confusion whether this means the header title or the main content title. Thoughts?

Reply:
> @cindyli - it should be the project name used for the alt text. Body title can be longer and more descriptive which won't be appropriate for alt text.
If so, another solution is to use the project name if it exists. Otherwise, use "body title". what do you think?

> I think "Body Title" should stay as it is because describes the location or function of that title (i.e. the title for the body of content). If it was just "title" there may be confusion whether this means the header title or the main content title. Thoughts?

It makes sense.

Reply:
In a conversation with @jhung in the channel, @jhung pointed out "Project Name" is a mandatory field. The issue is some content was authored before this requirement. So the old content should be updated to have a Project Name.

@chosww can you go through all project .md files and add "Project Name" if it's missing? We could use the same value in the body title for the project name. Thanks.

Reply:
Yep, I will do that :)